packages/core/src/elements/Accordion/FeAccordion/FeAccordionContext.ts

```diff
@@ -0,0 +1,8 @@
+import { createContext } from 'react';
+
+const defaultContext = {
+    expanded: false,
+    toggleExpanded: () => { }
```

Comment:
is this callback listener? pass the expanded state to the toggleExpanded function